geocoding/location-service/src/app/geocoder.py

```diff
@@ -76,7 +78,10 @@ def geocode(self, query, options={}):
         types = None
         if 'limitToResolution' in options:
             types = [self.resolutionToMapboxType(i) for i in options['limitToResolution']]
-        geoResult = mapbox_geocode(self.api_token, query, types=types, limit=5, languages=['en'])
+        countries = None
+        if 'limitToCountry' in options:
+            countries = options['limitToCountry']
```

Comment:
cosmetic, but this can be clubbed down to `countries = options.get("limitToCountry")`